lib/components/src/tree-list/tree-list-item.scss

```diff
@@ -0,0 +1,69 @@
+.itemContent {
+    // .itemContent is a div so its width will be the width of its parent.
+    padding: 0px 32px 0px var(--indent);
```

Comment:
The initial left padding/indent in an item should be 8px, not 16. After that each progressive indentation level adds 16, so level 2 is 24, level 3 is 40, etc

Reply:
I calculate the `--indent` value in `tree-list-item.ts` on line 40:

`style="--indent:${this.indentLevel*this.indentMultiple+8}px"`

I just changed 16 to 8 here.